### DISCUSSION TITLE
Download File .txt From Input Field

### DISCUSSION THREAD

**Discussion by hci-ikahera** [Budibase/budibase#11341] [Help] [answered]
Hello everyone,
Anyone here has any idea how to download file .txt from an input field using a button?
Thank you for any kind of help 🙏🏼

Regards,
Ika

**Comment by ConorWebb96** [comment]
Hey @hci-ikahera,

Could you provide a little bit more information on this. Can you explain a bit more in detail what you mean I'm unsure what you are after.

Are you wanting download the text entered as the value of an input field?

Thanks in advance for the information.

**Reply by hci-ikahera** [reply]
I have a text field which contains a value. I want to save this field value in the form of file .txt. Is it possible on Budibase?
Thank you.

**Reply by ConorWebb96** [reply, marked as answer]
Hey @hci-ikahera,

I don't believe this is possible natively in Budibase without doing some hacky methods using the embed component. The better route here would be to create a custom plugin which you can fully create/control that would achieve this functionality for you. You can find documentation on how to create custom plugins here; https://docs.budibase.com/docs/custom-component

Additionally, you can also see the various plugins that are already available here; https://github.com/budibase/plugins#components these can be used as references whenever making your own.

**Reply by AlejandroFNadal** [reply]
Perhaps this is useful @hci-ikahera 

https://github.com/Budibase/budibase/discussions/9032

I am not using this approach anymore and instead I have implemented a backend that writes the file to the Minio bucket of budibase and then generates a temporal download link. That is more complex but more versatile and it works in all browsers.
